pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -770,7 +799,48 @@ cdef class _Timedelta(timedelta):
 
     @property
     def resolution(self):
-        """ return a string representing the lowest resolution that we have """
+        """
```

Comment:
Isn't this already a part of #21122?

Reply:
@WillAyd 

Yes...
I am not sure why that is in there.
I created separate branches for each of the changes I am proposing to:

* delta
* resolution
* nanoseconds

etcetera, but the commit stream above seems to be sucking all of them into this PR.
super confused.

Reply:
Hmm this should just reflect what you have locally - any chance you merged your branches there accidentally?

Reply:
when creating a new branch, do it off of upstream/master (or if you keep a local master then off of this). each branch should be independent.

you will need to revert the resolution part of this change (as its in the other PR). or can just close that one and do it here is ok too)